applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/ProjectParameters.json

```diff
@@ -70,7 +70,7 @@
             "preconditioner_type": "ilu0"
         },
         "problem_domain_sub_model_part_list": ["bodemplaat"],
```

Comment:
Minor thing, may I suggest changing the problem name to English? I think it's better if they are consistent.